charts/velero/templates/backupstoragelocation.yaml

```diff
@@ -1,4 +1,43 @@
 {{- if .Values.backupsEnabled }}
+{{- if and .Values.configuration.backupStorageLocation (kindIs "slice" .Values.configuration.backupStorageLocation) }}
+{{- range .Values.configuration.backupStorageLocation }}
+---
+apiVersion: velero.io/v1
+kind: BackupStorageLocation
+metadata:
+  name: {{ .name }}
+  namespace: {{ $.Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade,post-rollback
+    "helm.sh/hook-delete-policy": before-hook-creation
+  labels:
+    app.kubernetes.io/name: {{ include "velero.name" $ }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+    helm.sh/chart: {{ include "velero.chart" $ }}
+spec:
+  credential: 
+    name: {{ .credential.name }}
+    key: {{ .credential.key }}
+  provider: {{ .provider }}
+  accessMode: {{ .accessMode }}
+  objectStorage:
+    bucket: {{ .bucket | quote }}
+    {{- with .prefix }}
+    prefix: {{ . | quote }}
+    {{- end }}
+    {{- with .caCert }}
+    caCert: {{ . }}
+    {{- end }}
+{{- with .config }}
```

Comment:
The default spec https://github.com/vmware-tanzu/helm-charts/blob/08bf56b0d8864014c1a7b704e3a3d35bc10848a0/charts/velero/templates/backupstoragelocation.yaml#L18 is missed, please add it back.

Reply:
Thanks very much for catching this!